README.md

```diff
@@ -109,6 +108,8 @@ Alternatively, you can specify all blob parameters exactly as you want by writin
             `free_parameter` for vx
 - `vy_parameter`: float, optional,
             `free_parameter` for vy
+- `spx_parameter`: float = 0.5,
```

Comment:
You forgot to change the variable name here. Also, remove trailing ,

Reply:
it is still spx_parameter in the constructor, thought of leaving it short since the others are also shortened (A_dist, etc) and changing them will make old code not work.

Reply:
Well, that's what I've done now anyway